### DISCUSSION TITLE
.gitkeep file is created

### DISCUSSION THREAD

**Discussion by okahijiki** [ConsenSys-archive/truffle#5584] [General Q&A] [answered]
After doing truffle init, I get the following instead of the normal file structure.
What should I do?

<img width="301" alt="スクリーンショット 2022-10-01 18 09 21" src="https://user-images.githubusercontent.com/22391336/193402274-3f896ba3-0c55-4f1b-8d38-5b1b3505ab75.png">

**Comment by LuisUrrutia** [comment, marked as answer]
That is the default structure of `truffle init`
The files `.gitkeep` allows git to track those folders even if they are empty.

If you want an "example" project, you must use `truffle unbox <box-name>`, u can see the docs over [here](https://trufflesuite.com/docs/truffle/getting-started/creating-a-project/)

<img width="234" alt="Captura de Pantalla 2022-10-01 a la(s) 22 26 57" src="https://user-images.githubusercontent.com/3771248/193434064-026d4771-5d2b-4f58-afbf-8d1aaecb199c.png">

**Reply by okahijiki** [reply]
Cleared. Thank you :)

**Reply by LuisUrrutia** [reply]
Cool, you should mark the question as answered in order to help other people with the same question.

**Reply by vcardozob** [reply]
Hey Luis, 

I'm following a tutorial in YT and I got to a point where after setting up some basic settings, the next step is to "truffle init" so I get the contracts, migrations, test folders as well as the truffle-config.js file. 

The problem is that I'm getting .gitkeep files instead of the actual files. Then I tried the "truffle unbox /path_of_the_folder" and had not success; any suggestion?
